Source/Main.cpp

```diff
@@ -149,9 +152,26 @@ int main(int argc, char *argv[])
                 MainMenu->normal_quit_button_instance->render();
             }
         }
-
-        frontHopper.render();
-        backHopper.render();
+        // Same block of if's as above but for pause screen
+        if (PauseMenu->IS_MENU_ACTIVE) {
+            // Start buttons
+            if (PauseMenu->render_start_shadow) {
+                PauseMenu->depp_start_button_instance->render();
+            } else {
+                PauseMenu->normal_start_button_instance->render();
+            }
+            // Quit buttons
+            if (PauseMenu->render_quit_shadow) {
+                PauseMenu->depp_quit_button_instance->render();
+            } else {
+                PauseMenu->normal_quit_button_instance->render();
+            }
+        }
+        // If GameObject variable start_game true
+        if (GameObject->start_game) {
```

Comment:
I'm also moving the game play loop functionality out into a different file, so this will get replaced soon, just FYI